Content.Shared/_RMC14/Weapons/Ranged/AmmoFixedDistanceComponent.cs

```diff
@@ -2,6 +2,13 @@
 
 namespace Content.Shared._RMC14.Weapons.Ranged;
 
-[RegisterComponent, NetworkedComponent]
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
 [Access(typeof(CMGunSystem))]
-public sealed partial class AmmoFixedDistanceComponent : Component;
+public sealed partial class AmmoFixedDistanceComponent : Component
+{
+    /// <summary>
+    /// If set to true, this makes shots from the weapon pass over creatures and objects.
+    /// </summary>
+    [DataField, AutoNetworkedField]
+    public bool HighArc = false;
+}
```

Comment:
How come this can't use a collision layer?

Reply:
Good question. Probably because it's thirty degrees in here and my brain is overheating. Fixing now.